### PR TITLE
Art oak

### PR DESCRIPTION
**Pull request by StefanCaldararu**
Add metapackage (basic for now)

### CODE REVIEW

**Comment by StefanCaldararu** [comment]
@AaronYoung5 git history problems bcs art-oak was behind master, had to merge. Lmk if you ahve a better fix for that...

File: atk.yml [hunk 1]
```diff
@@ -71,6 +71,21 @@ services:
         PIP_REQUIREMENTS: "${DEFAULT_PIP_REQUIREMENTS}"
         ROSDEP_METAPACKAGE: "art_art5_meta"
         ROSDEP_SKIP_KEYS: "python3-torchvision tf_transformations"
+  oak:
+    extends: dev
```
Comment (AaronYoung5) [inline atk.yml:75]: Can you create a common `agx` service that isn't meant to be run as a service, but instead has all the shared elements from this and `art5` (and the future other agx services). Does this make sense? Like really the only new stuff across each unique agx service is the `image`, `hostname`, `container_name`, and `ROSDEP_METAPACKAGE`.

File: workspace/src/common/meta/art_oak_meta/README.md
Comment (AaronYoung5) [inline workspace/src/common/meta/art_oak_meta/README.md:1]: Wanna write a small note? Whatever you like, just maybe not empty.

File: atk.yml (hunk 1 above)
Comment (StefanCaldararu) [reply atk.yml:75]: yessir

**Comment by StefanCaldararu** [comment]
@AaronYoung5 untested atk file, but does it look ok?

**Comment by AaronYoung5** [comment]
Ya looks good to me, I'd test first. I'm unable to rn.

**Review by AaronYoung5 — APPROVED** [review]

**Pull request merged**